src/main/java/ro/developmentfactory/thedrones/service/DronePositionService.java

```diff
@@ -0,0 +1,15 @@
+package ro.developmentfactory.thedrones.service;
+
+import org.springframework.transaction.annotation.Transactional;
+import ro.developmentfactory.thedrones.controller.dto.DroneStatusResponse;
+
+
+public interface DronePositionService {
+
+    @Transactional
+    DroneStatusResponse turningDirection(DroneStatusResponse droneStatusResponse);
+
+    @Transactional
+    DroneStatusResponse moveForward(DroneStatusResponse droneStatusResponse);
```

Comment:
you don't need to pass `DroneStatusResponse`. first of all, it's even not a request, it's a response. but 
- in order to turn, you need to know id of the drone and the direction. id you already have, so you can just pass it. for the direction you can use `TurnDirection` enum and pass it as a parameter
- in order to move forward you can just pass drone id and you have it from `@PathVariable` too